.eslintignore

```diff
@@ -3,3 +3,4 @@ docs
 go
 lib/*.js
 node_modules
+types
```

Comment:
I suggest uding `types/` instead to (a) only match the folder, not the file and (b) improve readbility as you see on first glance, this is an irgnored folder